### PR TITLE
VACMS-12250: drupal 9.5.2 upgrade

### LINKED ISSUE #12250: Upgrade to Drupal 9.5
## Description
Upgrade Drupal to version 9.5 in preparation for the upgrade to Drupal 10. 

## Acceptance Criteria
- [ ] Upgrade `drupal/core-recommended` to `~9.5.0`
- [ ] Upgrade `drupal/core-composer-scaffold` to `~9.5.0`

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Closes #12250.

## Testing done
Tested locally. Checked Staff Profile, Event, VAMC Detail page, Vet Venter and KB article with no errors. Also checked the Status Report, there were no unexpected errors or warnings. 

## Screenshots
CleanShot2023-01-19at08 30 50

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

1. Steve and/or Nate
   - [x] Please double check the removed and updated patches.
   - [x] Check Status report and Recent Logs for errors.
2. Laura and Erika
   - [ ] Please check content and any other areas you can think of for errors. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -364,29 +364,26 @@
...
                 "3027240 - Undefined index: #parents FormState.php": "https://www.drupal.org/files/issues/2019-01-21/form_system-undefined_index-3027240-2.patch",
-                "3096790 - Remove aria-required attribute": "https://www.drupal.org/files/issues/2021-09-28/drupal-remove-aria-require-3096790-7.patch",
+                "3096790 - Remove aria-required attribute": "https://www.drupal.org/files/issues/2022-12-16/3096790-30-9.5.patch",
                 "3219165 - Seven inactive vertical tabs do not meet contrast requirements": "https://www.drupal.org/files/issues/2021-06-16/3219165-3.patch",
                 "2915792 - MenuLinkContentAccessControlHandler does not allow \"view\" access without admin permission": "patches/drupal-core-menu-link-perm-mr-339.patch",
                 "2421145 - Text format wrapper does not take description_display into account": "https://www.drupal.org/files/issues/2021-12-03/2421445-18_0.patch",
-                "Internal Claro tweaks": "patches/claro-css-tweaks.patch",
```
Comment (ndouglas) [inline composer.json:384]: Is this obsolete or just not applying anymore?

**Comment by ndouglas** [comment]
```
  1) Step definitions function as expected
       The element with selector should have attribute:

      Timed out retrying after 10000ms
      + expected - actual

      -'contextualToolbar-init big-pipe-early-behaviors'
      +'contextualToolbar-init'
      
      at Context.eval (https://pr12283-ai3sywtubcy6umjfxgnacdeadexmggsd.ci.cms.va.gov/__cypress/tests?p=tests/cypress/integration/behavioral/step_definitions.feature:73221:19)
      at Context.resolveAndRunStepDefinition (https://pr12283-ai3sywtubcy6umjfxgnacdeadexmggsd.ci.cms.va.gov/__cypress/tests?p=tests/cypress/integration/behavioral/step_definitions.feature:27189:29)
      at Context.eval (https://pr12283-ai3sywtubcy6umjfxgnacdeadexmggsd.ci.cms.va.gov/__cypress/tests?p=tests/cypress/integration/behavioral/step_definitions.feature:26510:35)
```

might be related to the update and might need to be changed.

Otherwise, looks great.

**Review by ndouglas — DISMISSED** [review]

**Review by laflannery — COMMENTED** [review]
Only issue I found was regarding CSS, added to ticket here: #12250

File: composer.json (hunk 1 above)
Comment (edmund-dunn) [reply composer.json:384]: I was incorrect. I portion of that patch was obsolete. The rest was not. I had to reroll it.

**Review by laflannery — APPROVED** [review]

**Review by ndouglas — APPROVED** [review]
Looks right.

**Review by swirtSJW — APPROVED** [review]
Nice work on all this @edmund-dunn 
This all looks good. 

I also went though the changes back to 9.5 and all those changes seem like non-issues for us.
https://www.drupal.org/project/drupal/releases/9.5.0

I think we are good to go
image

**Pull request merged**